src/http/keys/keys.controller.ts

```diff
@@ -53,7 +53,7 @@ export class KeysController {
         try {
           for (const keysGenerator of keysGenerators) {
             for await (const keysBatch of keysGenerator) {
-              jsonStream.write(keysBatch);
+              jsonStream.write(JSON.stringify(keysBatch));
```

Comment:
Why do we need this change ?

Reply:
`JsonStream.stringify` returns `ReadWriteStream` which has write method. It requires string | Buffer | UInt8Array, but not object.

https://nodejs.org/api/stream.html#writablewritechunk-encoding-callback